app-misc/ani-cli/ani-cli-9999.ebuild

```diff
@@ -25,6 +29,6 @@ DEPEND="
 RDEPEND="${DEPEND}"
 
 src_install() {
-	dobin ani-cli
-	doman ani-cli.1
+	dobin "${DISTDIR}/ani-cli"
+	doman "${DISTDIR}/ani-cli.1"
```

Comment:
i don't think this will work with the 9999 ebuild. 😉

Reply:
Ohw , true .. okay , one sec.

Reply:
Done.